signing_service/signing_service/constants.py

```diff
@@ -38,3 +38,9 @@
 
 # Defines how much time in seconds should SigningService wait for AuthenticationChallengeFrame.
 RECEIVE_AUTHENTICATION_CHALLENGE_TIMEOUT = 20
+
+# Defines how many times Signing Service should try reconnecting on socket error, before giving up and crashing.
+SIGNING_SERVICE_DEFAULT_RECONNECT_ATTEMPTS = 5
```

Comment:
Why should it crash instead of constantly retrying?

Reply:
I have had an interesting chat with @cameel and we finally got to a conclusion, that there should be retry limit (and when it is exceeded, application should crash & report to `Sentry` to get some attention)

Reply:
> Why should it crash instead of constantly retrying?

We are not gone know when MM crashed if it just retrying. But 5 steps is to short of a time I think.